types/address.go

```diff
@@ -256,6 +264,31 @@ func (aa AccAddress) Format(s fmt.State, verb rune) {
 	}
 }
 
+// LengthPrefixedAddress prefixes the address bytes with its length, this is used
+// for variable-length components in store keys. Note: All addresses should be
+// max 255 bytes, or else this function panics.
+func LengthPrefixedAddress(bz []byte) ([]byte, error) {
+	if len(bz) == 0 {
```

Comment:
instead of repeating `len(bz)` 3 times we can create a var and use it bellow:
```
var bzLen = len(bz)
...
```

Reply:
Sure changed. I always assumed `len(bz)` is O(1) (cached), but is there a runtime overhead?

Reply:
No, it's not O(1). But the diff is negligible in this case - we call a function and read a variable from the slice struct. The diff is negligible. For me it's more of removing repeated function calls, which doesn't look good.

Reply:
I created a benchmark: https://gist.github.com/robert-zaremba/ea195543ec913186d80c305e84e38fac